### PR TITLE
En 1100 Servlet widget configuration

### PR DESCRIPTION
**Pull request by alessioscalici**
This includes:
- Widget configuration flow from Page configuration
- Necessary API switch from Promise mocks to apiManager makeRequest
- Internal servlet configuration form

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/ui/widgets/config/WidgetConfigPageContainer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export const mapDispatchToProps = dispatch => ({
+  onWillMount: () => dispatch(initWidgetConfigPage()),
+  onSubmit: (widgetConfig) => {
+    dispatch(updateConfiguredPageWidget(widgetConfig));
+  },
+});
+
+export const mapStateToProps = state => ({
+  widgetId: getParams(state).widgetCode,
+  framePos: parseInt(getParams(state).framePos, 10),
```
[the remark this answers is not in the document]
Comment (nicpuddu) [reply src/ui/widgets/config/WidgetConfigPageContainer.js]: we'll do some refactoring later on the router itself.

[the remark this answers is not in the document]
Comment (alessioscalici) [reply src/ui/widgets/config/WidgetConfigPageContainer.js]: We need to do a cast in the router package, Leaving a TODO comment

**Comment by entando-service** [comment]
All is well

**Pull request merged**